src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+// Types
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
As instructed by the previous reviewer the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:
const ADD_BOOK = bookstore/books/ADD_BOOK
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK

 and then you need to create a reducer that checks the action type using case, for example :

 switch (action.type) {
    case ADD_BOOK :
     ....
    case REMOVE_BOOK:
     ....
    default:
     ....
  }
kindly fix this for both books & categories